ansible/roles/kong-api/defaults/main.yml

```diff
@@ -76,7 +76,7 @@ object_category_definition_prefix: /object/category/definition
 dataset_service_prefix: /dataset
 asset_prefix: /asset
 collection_prefix: /collection
-
+discussions_prefix: /discussionHub
```

Comment:
Can you share this to discussion.
Chage in both middleware & here is the next release